packages/amplication-server/src/core/project/project.service.ts

```diff
@@ -106,6 +106,30 @@ export class ProjectService {
     return project;
   }
 
+  async createPreviewProject(
```

Comment:
This seems repetitive of the usual createProject method. I'm not sure it's really a big problem, but perhaps you can think of refactoring out some of the parts so you will keep DRY.

Reply:
The difference is the report usage, we don't need to report usage for the preview account and I don't want to add a pram "doReportUsage" or something like that

Reply:
Perhaps at least extract the common parts to utility methods so we won't have the same `prisma.create` statement twice, etc. (if the schema changes later, we would need to touch two places)